deploy/helm/flightctl/values.yaml

```diff
@@ -112,5 +112,3 @@ ui:
   enabled: true
   api:
     insecureSkipTlsVerify: true
-prometheus:
-  enabled: false
```

Comment:
oh we should have left a default here to 

prometheus:
    enabled: false

otherwise some charts fail when deploying on openshift